client/src/rpc_request.rs

```diff
@@ -38,7 +38,6 @@ pub enum RpcRequest {
     GetVersion,
     GetVoteAccounts,
     RegisterNode,
-    RequestAirdrop,
```

Comment:
Do we need to keep a placeholder here for the enum value or does everyone specify the method with a string?

Reply:
It's all string based so this won't harm anything